nixops/backends/__init__.py

```diff
@@ -329,6 +337,20 @@ def send_keys(self) -> None:
             )
             self.run_command("mv " + tmp_outfile_esc + " " + outfile_esc)
             os.remove(tmp)
+
+        def task(key: str, opts: Dict[str, Any]) -> nixops.parallel.Task:
+            task = types.SimpleNamespace()
```

Comment:
is there any concern with having two things named `task` here?

Reply:
fixed